.gitcommit_pulltemplate.txt

```diff
@@ -0,0 +1 @@
+jackett indexers as of 
```

Comment:
Is this necessary

Reply:
the language or the file?

the file is not as it can be  hardcoded in the script

the language yes - so we can identify the previous commit we pulled

Reply:
The file